web/src/features/fba/components/FBAMap.tsx

```diff
@@ -65,10 +45,30 @@ const FBAMap = (props: FBAMapProps) => {
   const [map, setMap] = useState<ol.Map | null>(null)
   const mapRef = useRef<HTMLDivElement | null>(null)
 
-  const fireCenterSource = buildFireVectorSource(fireCenterLayer)
+  const fireZoneVector = new VectorTileLayer({
+    // declutter: true,
+    opacity: 0.5,
+    source: new VectorTileSource({
+      attributions: 'BC stuff',
+      format: new MVT(),
+      url: 'https://tileserv-dev.apps.silver.devops.gov.bc.ca/public.fire_zones/{z}/{x}/{y}.pbf'
```

Comment:
@conbrad can we make this configurable? Thinking we'll have a dev and prod tileserver.

Reply:
Do we have a frontend configuration setup already? Not familiar with how we do this on frontend.

Reply:
As discussed offline, let's defer this change in a future PR.